fw/charybdis-features.md

```diff
@@ -56,72 +58,115 @@ For each mode, the firmware allows cycling through multiple pre-defined values:
-- Sniping DPI Decrease (`Snp-`): decrease the sensitivity of the pointer movement in sniping mode by one step (out of 4).
-- Sniping Momentary (`Snp`): enable sniping mode as long as the key is pressed.
-- Sniping Toggle (`SnpT`): toggle sniping mode on and off.
-- Drag-scroll Momentary (`Drg`): enable drag-scroll mode as long as the key is pressed.
-- Drag-scroll Toggle (`DrgT`): toggle drag-scroll mode on and off.
+### Sniping
 
-### Charybdis 4x6
+**Sniping mode** slows down the pointer for more precise gestures. It is useful when combined with a higher default DPI. Like the default pointer's DPI, the sniper mode DPI can be changed at runtime
 
-- 3x5: heavily inspired by Miryoku
-  - Base layer
-  - Numbers layer
-  - Symbols layer
-  - Function layer
-  - Navigation layer
-  - Media layer
-  - Pointer layer
-- 4x6: inspired from OG dactyl
-  - Base layer
-  - Lower layer
-  - Raise layer
-  - Pointer layer
+Custom Keycodes:
 
-## Firmware configuration
+| Name   | Description                                                                  |
+| ------ | ---------------------------------------------------------------------------- |
+| `S_D_MOD` | increase the sensitivity of the pointer movement in sniping mode by one step |
+| `S_D_RMOD` | decrease the sensitivity of the pointer movement in sniping mode by one step |
+| `SNIPING`  | enable sniping mode as long as the key is pressed                            |
+| `SNP_TOG` | toggle sniping mode on and off                                               |
 
-{: .note }
-This requires a firmware update.
 
-The following section explains how to update the firmware source code to fine-tune your experience. This requires a functional development environment for QMK (https://docs.qmk.fm/#/newbs) and, ideally, some prior experience in computer science and the C programming language.
+Custom defines (with default values):
 
-### Changing dynamic DPI scaling default and increment values
+```c
+#define CHARYBDIS_MINIMUM_SNIPING_DPI 200
+#define CHARYBDIS_SNIPING_DPI_CONFIG_STEP 100
+```
 
-{: .note }
-This requires a firmware update.
+Custom functions:
+
+```c
+charybdis_set_pointer_sniping_enabled(bool enable) // enable/disable sniping mode
+charybdis_get_pointer_sniping_enabled() // returns whether sniping mode is currently enabled
+charybdis_cycle_pointer_sniping_dpi(bool forward) // cycle forward or backward the possible values
+charybdis_cycle_pointer_sniping_dpi_noeeprom(bool forward) // cycle forward or backward the possible values without persisting the change to EEPROM
+charybdis_get_pointer_sniping_dpi() // returns the current sniping mode DPI value
+```
 
 ### Auto sniping on layer
 
-{: .note }
-This requires a firmware update.
+You can trigger sniping automatically when on a specific layer by adjusting the following in your keymap:
+
+```c
+#define CHARYBDIS_AUTO_SNIPING_ON_LAYER LAYER_POINTER
+```
 
 ### Auto pointer layer
```

Comment:
Should we also mention core auto mouse?

Reply:
I also vote on keeping things simple here, in the "bkb ecosystem"